Create ui-c8y-1021-37-5-unrelated-alarms-not-shown-in-alarmstab-8113-graft-release-cd-8260.md (#2924)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-37-5-unrelated-alarms-not-shown-in-alarmstab-8113-graft-release-cd-8260.md

---
date: ""

> title: Unrelated alarms are no longer shown in the simulators alarms tab.

Reviewer: ```suggestion
title: Alarms tab in the Simulator page only shows alarms related to the selected asset
```

> title: unrelated alarms not shown in alarmstab (#8113) [GRAFT][release/cd] (#8260)

Reviewer 2: ```suggestion
title: Unrelated alarms are no longer shown in the simulators alarms tab.
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62196
version: 1021.37.5
---

> unrelated alarms not shown in alarmstab (#8113) [GRAFT][release/cd] (#8260)

Reviewer 2: ```suggestion
In the simulators page, the alarms tab previously displayed alarms that were not directly related to the currently selected asset. This caused confusion for users who expected to only see relevant alarms for the specific asset they were viewing. With this change, the alarms tab now correctly filters out unrelated alarms, ensuring that users only see alarms that are pertinent to the selected asset. This improvement enhances clarity and usability when monitoring and managing alarms within the application.
```

> In the simulators page, the alarms tab previously displayed alarms that were not directly related to the currently selected asset. This caused confusion for users who expected to only see relevant alarms for the specific asset they were viewing. With this change, the alarms tab now correctly filters out unrelated alarms, ensuring that users only see alarms that are pertinent to the selected asset. This improvement enhances clarity and usability when monitoring and managing alarms within the application.

Reviewer: ```suggestion
In the **Simulator** page, the **Alarms** tab previously displayed alarms that were not directly related to the selected asset. This caused confusion for users who expected to only see relevant alarms for the specific asset they were viewing. With this change, the **Alarms** tab now correctly filters out unrelated alarms, ensuring that users only see alarms that are pertinent to the selected asset.
```
